tests/common/fixtures/advanced_reboot.py

```diff
@@ -123,8 +125,9 @@ def __buildTestbedData(self):
         self.rebootData['vlan_ip_range'] = self.mgFacts['minigraph_vlan_interfaces'][0]['subnet']
         self.rebootData['dut_vlan_ip'] = self.mgFacts['minigraph_vlan_interfaces'][0]['addr']
 
-        self.rebootData['dut_username'] = self.creds['sonicadmin_user']
-        self.rebootData['dut_password'] = self.creds['sonicadmin_password']
+        hostvars = self.duthost.host.options['variable_manager']._hostvars[self.duthost.hostname]
+        self.rebootData['dut_username'] = Template(self.creds['sonicadmin_user']).render(**hostvars)
```

Comment:
can the template be done once in the creds fixture itself?

Reply:
I believe fixture can be a class. But let's move on to fix the issue then improve.